tests/Aspire.Hosting.Tests/ApplicationModel/ReferenceExpressionTests.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace Aspire.Hosting.Tests.ApplicationModel;
+public class ReferenceExpressionTests {
```

Comment:
Can you make the formatting match the rest of the project?

@eerhardt @joperezr Do we have an editor config?

Reply:
There is one in the root. I was making these changes on VSCode on Linux and missed the brace not on the newline. I'll check it out on VS on Windows and make sure all of the editorconfig rules are met.

Reply:
Updated with correct formatting.